src/sentry/api/endpoints/organization_tagkey_values.py

```diff
@@ -36,15 +38,16 @@ def get(self, request, organization, key):
                 ('count()', '', 'count'),
             ],
             orderby='-count',
-            groupby=['tags_value'],
+            groupby=[column_name],
```

Comment:
i think this and also the use below need to both be conditionals like `column_name if is_column else 'tags_value'`